lib/rules/prefer-template.js

```diff
@@ -187,6 +201,17 @@ module.exports = {
                 return sourceCode.getText(currentNode);
             }
 
+            if (isConcatenation(currentNode) && !hasIdentifierReference(currentNode)) {
+
+                /*
+                 * As there are no varibales used in this concatenation, we can sipmly calculate the concatenated string
+                 * "'Hello ' + 'world'" -> ["'Hello ' " , " 'world'"] -> ["Hello ", "world"] -> "Hello world"
+                 */
+                const concatenatedText = sourceCode.getText(currentNode).split("+").map(e => e.trim().slice(1, -1)).join("");
+
+                return `\`${concatenatedText}\``;
```

Comment:
After thinking about this again, ignoring those chains would be a big change in the functionality of this rule, so it might be better to just improve the autofix.